frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/utils.tsx

```diff
@@ -0,0 +1,102 @@
+import * as _ from 'lodash';
+import { Humanize } from '@console/internal/components/utils';
+import { PrometheusResponse } from '@console/internal/components/graphs';
+import { Colors } from './bar-colors';
+
+const getTotal = (stats: StackDataPoint[]) => {
+  return stats.reduce((total, dataPoint) => total + dataPoint.y, 0);
+};
+
+const addOthers = (stats: StackDataPoint[], totalUsed: string, formatValue: Humanize) => {
```

Comment:
```suggestion
const addOthers = (stats: StackDataPoint[], totalUsed: string, humanize: Humanize) => {
```
I think this is the convention through the console. For other instances also.

Reply:
ack